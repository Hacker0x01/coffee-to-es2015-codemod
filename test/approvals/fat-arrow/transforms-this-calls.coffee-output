var handleChange;

handleChange = (function(_this) {
  return function(event) {
    return console.info(_this.foo);
  };
})(this);
