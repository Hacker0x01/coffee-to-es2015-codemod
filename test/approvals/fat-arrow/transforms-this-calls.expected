var handleChange;

handleChange = event => {
  return console.info(this.foo);
};
